site/content/blogs/release-v122.en.md

```diff
@@ -38,23 +38,23 @@ Copilot v1.22 brings several new features and improvements:
 
 ## CloudFront TLS Termination
 
-You can now configure env manifest to have CloudFront terminate TLS for your Load Balanced Web Services (LBWS):
+You can now configure your env manifest to have CloudFront terminate TLS for your Load Balanced Web Services (LBWS):
 
 ```yaml
 cdn:
   tls_termination: true
 ```
 
-The configuration above uses CloudFront for TLS termination, which means the traffic from `CF → ALB → ECS` will be HTTP only. This brings faster TLS termination and shorter page loading for viewers, since the CloudFront edges are usually geographically closer to the them.
+The configuration above uses CloudFront for TLS termination, which means the traffic from `CF → ALB → ECS` will be HTTP only. This brings faster TLS termination and shorter page loading for viewers, since the CloudFront edges are usually geographically closer to them.
 
-However, if your services have HTTPS enabled (either you have an app domain or imported certificates in the environment), you must turn off ALB http redirect by updating your LBWS manifests.
+However, if your services have HTTPS enabled (you have either an app domain or imported certificates in the environment), you must turn off ALB http redirect by updating your LBWS manifests.
```

Comment:
I don't know if we wrote LBWS anywhere before 😂 
```suggestion
However, if your services have HTTPS enabled (you have either an app domain or imported certificates in the environment), you must turn off ALB http redirect by updating your [Load Balanced Web Service manifests](path/to/manifest link).
```

Reply:
yeah it's actually here https://github.com/aws/copilot-cli/pull/4013/files#diff-04ecc586aa589f72329c52799921229fc5ff7e898637bf1c965f2ea4a15a7199R41